spec/setup_acceptance_node.pp

```diff
@@ -15,14 +15,17 @@
 
 # Necessary for PostgreSQL EVR extension
 yumrepo { "pulpcore":
-  baseurl  => "http://yum.theforeman.org/pulpcore/3.14/el\$releasever/x86_64/",
+  baseurl  => "http://yum.theforeman.org/pulpcore/nightly/el\$releasever/x86_64/",
   descr    => "Pulpcore",
   enabled  => true,
-  gpgcheck => true,
-  gpgkey   => "https://yum.theforeman.org/pulpcore/3.14/GPG-RPM-KEY-pulpcore",
```

Comment:
Hey @zjhuntin , not sure if this was intentional or not, but there's no GPG key in the nightly RPM builds for pulpcore.

It doesn't really matter for the purposes of this PR, since this is for CI and we'd be fetching the GPG key from the same location as the RPMs, using it once, and then tearing it down, but I wanted to give you as a heads up just in case it matters more to somebody somewhere else. ;-)